module/User/Module.php

```diff
@@ -39,10 +39,10 @@ public function onBootstrap(MvcEvent $e)
 
         // this event listener will turn the request into '403 Forbidden' when
         // there is a NotAllowedException
-        $em->attach(MvcEvent::EVENT_DISPATCH_ERROR, function($e) {
+        $em->attach(MvcEvent::EVENT_DISPATCH_ERROR, function ($e) {
             if (($e->getError() == 'error-exception') &&
-                    ($e->getParam('exception', null) != null) &&
-                    ($e->getParam('exception') instanceof NotAllowedException)) {
+                ($e->getParam('exception', null) != null) &&
+                ($e->getParam('exception') instanceof NotAllowedException)) {
```

Comment:
Perhaps we should do this as described in the second example of [PSR-12 5.1](https://www.php-fig.org/psr/psr-12/#51-if-elseif-else). Improves the readability.

Reply:
Good suggestion